### DISCUSSION TITLE
Progress meter for long running ops

### DISCUSSION THREAD

**Discussion by stelmo** [LCSB-BioCore/COBREXA.jl#629] [Ideas]
Sampling takes long, I would like to know how long I need to wait... Something like https://github.com/timholy/ProgressMeter.jl would be nice...

**Comment by exaexa** [comment]
Let's generalize a bit, this is also useful when planning a coffee pause during a FVA or screening. :D

**Comment by exaexa** [comment]
looking at it, this would probably require a `pmap()` version that supports the progress reporting
